flink-end-to-end-tests/test-scripts/test_kubernetes_embedded_job.sh

```diff
@@ -37,10 +39,28 @@ function cleanup {
     rm -rf ${OUTPUT_VOLUME}
 }
 
+function check_kubernetes_status {
```

Comment:
I'm guessing these work because the return value of a bash function is implicitly the return value of the last executed command?

Maybe we can make that more explicit so that we don't accidentally break it in the future if someone adds another line after the last line. (This is also something I don't like that much about Scala ...)